SU-HSDO/stanford_mrc
# Mrcd8 253 media views

# READY FOR REVIEW

# Summary
- Changed all fields in views to use the new media fields.
- delete old fields after views revert.
- Delete migrate data if entity is deleted.
- Corrected form display settings
- Cleaned up field save functionality and put it in the service class
- Abstracted field formatter for media
- new field formatter for media for responsive image style
- Add permissions for site owners for the new media browsers

# Needed By (Date)
- End of sprint

# Urgency
- High

# Steps to Test

1. Sync to prod
2. checkout this branch and update database
3. Click around on the site and ensure no errors or missing images/views
4. edit/create some content with the media fields.
5. ensure they look as expected.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
